docs/reference/2-protocols/1-coap/6-rpc.md

```diff
@@ -0,0 +1,95 @@
+---
+id: rpc
+title: RPC
+---
+
+[Remote Procedure Call (RPC) Service](/device-management/rpc) definitions over
+CoAP.
+
+How to use guides:
+
+- [RPC Overview](/device-management/rpc)
+
+## Interface
+
+| Method  | Description                        | Path              | Content Format |
+| ------- | ---------------------------------- | ----------------- | -------------- |
+| OBSERVE | Listen for RPCs sent to a device.  | `/.rpc`           | JSON/CBOR      |
+| POST    | Report RPC status and return data. | `/.rpc/status`    | JSON/CBOR      |
+
+## RPC Parameters and attributes:
+
+Here is an example of an RPC payload sent to a device by the Golioth server:
+
+```json
+{
+  "id": "multiply/74e4",
+  "method": "multiply",
+  "params": [ 7, 191 ]
+}
+```
+
+| Attribute | Description                                                  |
+| --------- | ------------------------------------------------------------ |
+| `id`      | String representation of a unique id assigned by the server. |
+| `method`  | String indicating the function name called.                  |
+| `params`  | Array of input parameters to pass to method.                 |
+
+The `params` may be empty, or contain any combination of values in the following
+formats:
+
+- `boolean`
+- `float`
+- `integer`
+- `string`
+
+## RPC Status Parameters and attributes:
+
+The device returns a payload Golioth server what includes the status of the RPC
+and any data returned by the function:
+
+```json
+{
+  "id": "multiply/74e4",
+  "status": 0,
+  "detail": {
+    "value": 1337
+  }
+}
+```
+
+| Attribute | Description                                                                                              |
+| --------- | -------------------------------------------------------------------------------------------------------- |
+| `id`      | String representation of the unique id received from the server.                                         |
+| `status`  | Integer [status code](/reference/protocols/coap/rpc#rpc-status-codes) indicating the outcome of the RPC. |
+| `detail`  | Array of output parameters.                                                                              |
+
+The `detail` may be empty, or contain any combination of key/value pairs in the
+following formats:
+
+- `boolean`
+- `float`
+- `integer`
+- `string`
+
+## RPC Status Codes
```

Comment:
Would it make sense to link to firmware SDK docs as we do in device management to make sure this list does not become stale?